src/main/java/uk/gov/hmcts/reform/em/annotation/service/dto/TagDTO.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.em.annotation.service.dto;
+
+import javax.validation.constraints.Size;
+import java.util.UUID;
+
+/**
+ * A DTO for the Tag entity.
+ */
+public class TagDTO {
+
+    private UUID id;
+
+    @Size(max = 35)
+    private String name;
+
+    private String createdBy;
+
+    @Size(max = 20)
+    private String label;
+
+    private String color;
+
+    private UUID annotationId;
```

Comment:
as above, not sure why we need the annotation on the tag

Reply:
As above, this is required to map between tags and annotation

Reply:
yes, but I thought it's one direction. annotations need to reference tags, but I don't get why tags need to know which annotations they're part of